app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(name=person["name"], age=person["age"])
+                   for person in people]
```

Comment:
Here you are iterating over the `people` list twice: once to create the `Person` objects and another to assign husbands and wives. This is inefficient and can be slow for large lists. Consider doing everything in a single loop.